### PR TITLE
Add Contributor ActionCreator and Store test

### LINKED ISSUE #717: Add Contributor ActionCreator test
## Kind (Required)

- Other

 ## Overview (Required)

- You can see other screen tests for your reference.

 ## Links	

- ContributorActionCreator https://github.com/DroidKaigi/conference-app-2019/blob/257fc9f4d3a13ae8f968b80e1878b41d30e45598/feature/contributor/src/main/java/io/github/droidkaigi/confsched2019/contributor/action/ContributorActionCreator.kt
- You can see AnnouncementActionCreatorTest https://github.com/DroidKaigi/conference-app-2019/blob/master/feature/announcement/src/test/java/io/github/droidkaigi/confsched2019/announcement/ui/actioncreator/AnnouncementActionCreatorTest.kt#L21

### LINKED ISSUE #722: Add Contributor Store test
## Kind (Required)

Choose the suitable template if exists.

- Other

 ## Overview (Required)

- You can see other screen tests for your reference.

 ## Links	

- ContributorStore https://github.com/DroidKaigi/conference-app-2019/blob/257fc9f4d3a13ae8f968b80e1878b41d30e45598/feature/contributor/src/main/java/io/github/droidkaigi/confsched2019/contributor/store/ContributorStore.kt
- AnnouncementStoreTest https://github.com/DroidKaigi/conference-app-2019/blob/master/feature/announcement/src/test/java/io/github/droidkaigi/confsched2019/announcement/ui/store/AnnouncementStoreTest.kt#L20

### PR DESCRIPTION
**Pull request by rince**
## Issue
- close #717 
- close #722 

## Overview (Required)
- Add Contributor ActionCreator test
  - Rename package from action to actioncreator
  - Fix typo on comment
- Add Contributor Store test

### CODE REVIEW

**Comment by rince** [comment]
@takahirom 
Sorry for my late PR, please check whenever you have time :bow:

**Comment by takahirom** [comment]
Thanks. Please wait 🙇

**Comment by takahirom** [comment]
LGTM 👍

File: feature/contributor/src/main/java/io/github/droidkaigi/confsched2019/contributor/actioncreator/ContributorActionCreator.kt [hunk 1]
```diff
@@ -19,7 +19,7 @@ class ContributorActionCreator @Inject constructor(
 ) : CoroutineScope by lifecycle.coroutineScope, ErrorHandler {
     fun load() = launch {
         try {
-            // Load Local cashe
+            // Load Local cache
```
Comment (takahirom) [inline feature/contributor/src/main/java/io/github/droidkaigi/confsched2019/contributor/actioncreator/ContributorActionCreator.kt]: You have eagle eyes 🦅

**Pull request merged**